ddtrace/internal/periodic.py

```diff
@@ -228,6 +228,6 @@ def join(
     def on_shutdown():
         pass
 
-    @staticmethod
-    def periodic():
+    def periodic(self):
```

Comment:
unrelated?

Reply:
well it is otherwise mypy complains :)

Reply:
Ah missed that a typehint was added in the test case :+1: